docs/customising/themes.md

```diff
@@ -65,11 +70,10 @@ getting the git repo from the URL specified in the config setting
 [`THEME_URLS`]({{ page.baseurl }}/docs/customising/config/#theme_urls). This is
 why your theme must be in its own git repo.
 
-One way to create your own theme is to fork the `alavetelitheme` theme from
-[https://github.com/mysociety/alavetelitheme](https://github.com/mysociety/alavetelitheme)
-(giving it your own theme name), edit it or add files, and deploy it with `themes:install`.
-Alternatively, since your site already has the default theme's files within it,
-you can duplicate `alivetelitheme` (in `lib/themes/`) and change its name.
+<div class="attention-box helpful-hint">
```

Comment:
:grey_question: undecided about this... having 2 attention boxes next to each other cancels them out a little bit; but on the other hand - probably a good place to "repeat" the new theme advice/link for people who have jumped to this heading